### PR TITLE
Testing script v2 update schema

### PR DESCRIPTION
**Pull request by skyfishjy**
- Update schema
- Add arguments (publish-username,--publish-password)

### CODE REVIEW

File: script/testing/oltpbench/reporting/parse_data.py [hunk 1]
```diff
@@ -1,8 +1,10 @@
 #!/usr/bin/python3
 
 import os
+import platform
```
Comment (benliangw) [inline script/testing/oltpbench/reporting/parse_data.py:4]: Did we use this package?

File: script/testing/oltpbench/test_case_oltp.py [hunk 2]
```diff
@@ -117,7 +119,7 @@ def run_post_test(self):
             # publish results
             if self.publish_results:
                 report(self.publish_results, os.path.join(
-                    os.getcwd(), "oltp_result",self.filename_suffix))
+                    os.getcwd(), "oltp_result",self.filename_suffix),self.publish_username,self.publish_password,self.query_mode)
```
Comment (benliangw) [inline script/testing/oltpbench/test_case_oltp.py:122]: Maybe you can use pep8 and flake8 to check the format. I think they can automatically fix the formatting issues.

Comment (benliangw) [reply script/testing/oltpbench/test_case_oltp.py:122]: Sorry, I just realized we don't have pep8 in this repo right?

File: script/testing/oltpbench/reporting/parse_data.py (hunk 1 above)
Comment (bialesdaniel) [reply script/testing/oltpbench/reporting/parse_data.py:4]: line 20

**Review by bialesdaniel — APPROVED** [review]

**Pull request merged**